script.js

```diff
@@ -0,0 +1,54 @@
+const menuIcon = document.getElementById('menu-icon');
+const xIcon = document.getElementById('x-icon');
+const header = document.getElementById('header');
+const descNav = document.getElementById('desc-nav');
+const logo = document.getElementById('logo');
+const listItems1 = document.getElementById('list-items-1');
+const listItems2 = document.getElementById('list-items-2');
+const listItems3 = document.getElementById('list-items-3');
+
+function myFunction(x) {
```

Comment:
While you did implement the previous reviewer's requirements, you didn't do so properly, below you will find a list of the requirements the project is still missing.

<img width="665" alt="Screen Shot 2022-07-20 at 18 23 18" src="https://user-images.githubusercontent.com/82426682/180099073-31244d67-905a-422c-a43b-b4a9a646b37f.png">

When I click on the `Portfolio` link I am not taken to the correct part of the page and the menu is not closing itself after I click on any of the links available. Please fix these issues.